src/otx/core/exporter/native.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Class definition for native model exporter used in OTX."""
+
+from __future__ import annotations
+
+import tempfile
+from pathlib import Path
+from typing import Any
+
+import onnx
+import openvino
+import torch
+
+from otx.core.exporter.base import OTXModelExporter
+from otx.core.types.export import OTXExportPrecisionType
+
+
+class OTXNativeModelExporter(OTXModelExporter):
+    """Exporter that uses native torch and OpenVINO conversion tools."""
+
+    def __init__(
+        self,
+        input_size: tuple[int, ...],
```

Comment:
```suggestion
        input_size: tuple[int, int],
```
I think input size always integer

Reply:
`...` here denotes any number of ints